packages/user/src/views/VerifyEmail.tsx

```diff
@@ -0,0 +1,72 @@
+import { useTranslation } from "@dzangolab/react-i18n";
+import { Page } from "@dzangolab/react-ui";
+import { Card } from "primereact/card";
+import { useContext, useEffect, useState } from "react";
+import { toast } from "react-toastify";
+
+import verifyEmail from "@/supertokens/verifyEmail";
+
+import { UserContextType, useUser, userContext } from "..";
+
+const VerifyEmail = () => {
+  const { t } = useTranslation("user");
+  const [verifyEmailLoading, setVerifyEmailLoading] = useState<boolean>(false);
+  const [message, setMessage] = useState<string>("");
```

Comment:
instead of common `message` state for all responses, having separate boolean states to represent each response would give us more flexibility in terms of UI. We could use the boolean states whichever way we want, here to display different messages for different states.